### DISCUSSION TITLE
1.1.0 지도상 새로고침 기능 이용시 시간업데이트 문제

### DISCUSSION THREAD

**Discussion by ktm0503** [changdoc/naver-vaccine-macro#134] [Q&A]
네이버지도상에서 새로고침 기능을 사용하였을때 업데이트 버튼은 계속 눌려지지만
현재시간이 아닌 과거시간에 멈춰있거나 현재시간과 딜레이가 발생합니다.
이렇게보여도 정상작동은 되는것인지 궁금합니다.

**Comment by changdoc** [comment]
시간 갱신은 어떻게 이루어지는지 모르겠는데 제가 테스트했을때는 분단위로 잘 갱신이 됐었네요. 
어떤 문제인지 예상되는 부분이 없는데... 개발자도구에서 console창에 에러 같은게 떴을까요?

**Reply by wuhancorona** [reply]
![image](https://user-images.githubusercontent.com/88173332/127599933-df0c99e2-201d-44c7-9bfa-0175c691d311.png)

정확히는 측정 안 해봤는데 20~40분 정도 하면 이렇게 오류가 뜹니다. 시간 갱신이 안 되고

이 상태에서 새로고침을 해도 저기에 오류가 뜹니다. 사람이 안 몰릴 땐 계속 냅두면 다시 5분쯤

뒤에 풀리는데 몰리는 시간엔 30분을 냅둬도 안 되네요. 해결 방법은 쿠키 지우고 다시 하면 되는데

정확히는 크롬 기준으로 naver.com 쿠키만 지워도 잘 됩니다. www.naver.com 쿠키, nid.naver.com 쿠키,

m.place.naver.com 쿠키 등은 냅둬도 되더라구요. 저는 개발자가 아니라서 정확히는 잘 모르겠네요 ㅠㅠ

**Reply by changdoc** [reply]
이상하네요. 저는 몇시간 켜둬도 이슈가 발생하지 않던데요.
크롬에서는 정상이면 크롬으로 진행해보시는건 어떨까요?

**Reply by wuhancorona** [reply]
크롬에서 정상이 아니고, 크롬 기준으로 저렇게 쿠키 하나만 지우고 다시 재로그인과 인증서 인증을 해야 하구요. 웨일이나 엣지는 그냥 쿠키 비우기를 해서 모르겠는데 크롬과 동일하게 몇 분 정도 하면 시간 갱신이 안 되어서 저렇게 됩니다. 똑같이 쿠키 지우고 재로그인이랑 인증서 인증을 해야 합니다. 이 프로그램 말고 현재 유튜브에서 유명한 영상의 매크로로도 똑같이 돌려놓고 시간 조금 지나면 저렇게 됩니다.
